anyway/views/news_flash/api.py

```diff
@@ -25,14 +51,14 @@ def news_flash():
     road_segment = request.values.get("road_segment_only")
     offset = request.values.get("offset", DEFAULT_OFFSET_REQ_PARAMETER)
     limit = request.values.get("limit", DEFAULT_LIMIT_REQ_PARAMETER)
-    news_flash_obj = db.session.query(NewsFlash)
-    news_flash_obj.offset(offset)
+    query = session.query(NewsFlash)
+    query.offset(offset)
```

Comment:
Did you mean `query = query.offset(offset)`?
*I think @atalyaalon has touched this code in her recent pull request